drivers/NFSSR.py

```diff
@@ -137,19 +137,24 @@ def attach(self, sr_uuid):
         if not self._checkmount():
             self.validate_remotepath(False)
             util._testHost(self.dconf['server'], NFSPORT, 'NFSTarget')
-            io_timeout = util.get_nfs_timeout(self.session, sr_uuid)
-            self.mount_remotepath(sr_uuid, io_timeout)
+
+            #Extract timeout and retrans values, if any
+            sr_other_config = util.get_other_config(self.session, sr_uuid)
```

Comment:
I meant a property for the SR called other_config that would have been initialized to None in the constructor and
populated the very first time a function needed it.

I think with this patch it is ok to go straight to the point, so keep this and, maybe, we can change things later